client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -118,6 +118,12 @@ export default React.createClass( {
 						{ this.props.message }
 					</span>
 				</div>
+				{ ( this.props.event || this.props.feature ) &&
```

Comment:
Do we need these? If so, I would propose we set `event` as required in the `propTypes`... which would require mucking about in the example docs, so that's probably fine as a separate PR.